src/helpers/strings.ts

```diff
@@ -14,8 +14,9 @@ export class Strings {
     static NavigateToBuildSummary: string = "Click to view build";
     static NavigateToTeamServicesWebSite: string = "Click to view your team project website.";
     static NoAccessTokenFound: string = "A personal access token for this Team Services repository was not found in your local user settings.";
-    static NoAccessTokenRunLogin: string = "No personal access token was found for Team Services.  Please run the 'team login' command.";
-    static NoTeamServerCredentialsRunLogin: string = "No credentials were found for Team Foundation Server.  Please run the 'team login' command.";
+    static NoAccessTokenLearnMoreRunLogin: string = "You are not connected to Team Services. Please learn more and then run the 'team login' command.";
```

Comment:
I don't think it will mean that clicking the button is all they have to do, because it says "and then run the 'team login' command."  I can see myself missing the button, and "please learn more" doesn't tell me how to learn more.